app/frontend/actions/reloadWallet.ts

```diff
@@ -26,13 +26,15 @@ export default (store: Store) => {
     const previousWalletReloadTime = _lastWalletReloadTime
     try {
       updateLastReloadTime()
-      const accountsInfo = await wallet.getAccountsInfo(state.validStakepoolDataProvider)
-      const tokensMetadata = await wallet.getTokensMetadata(accountsInfo)
+      const accountsInfo = state.validStakepoolDataProvider
+        ? await wallet?.getAccountsInfo(state.validStakepoolDataProvider)
```

Comment:
`getWalletOrThrow()` should ensure that the wallet is defined
```suggestion
        ? await wallet.getAccountsInfo(state.validStakepoolDataProvider)
```